src/components/calcite-slider/calcite-slider.scss

```diff
@@ -268,11 +271,11 @@
 }
 
 .tick__label--min {
-  transition: opacity 150ms;
+  transition: opacity var(--calcite-internal-animation-timing);
 }
 
 .tick__label--max {
-  transition: opacity 50ms;
+  transition: opacity calc(50ms * var(--calcite-animation-timing-factor));
```

Comment:
Since there's only a 50ms difference between the internal global vars, could this use either the `quick` var? cc @macandcheese

Reply:
Yeah let's standardize to use the established ones here.